RecoLocalTracker/SiPixelClusterizer/plugins/SiPixelRawToClusterCUDA.cc

```diff
@@ -105,15 +109,17 @@ SiPixelRawToClusterCUDA::SiPixelRawToClusterCUDA(const edm::ParameterSet& iConfi
 
   edm::Service<CUDAService> cs;
   if (cs->enabled()) {
-    wordFedAppender_ = std::make_unique<pixelgpudetails::SiPixelRawToClusterGPUKernel::WordFedAppender>();
+    wordFedAppender_ = std::make_unique<pixelgpudetails::SiPixelRawToClusterGPUKernel::WordFedAppender>(maxFedWords_);
   }
 }
 
 void SiPixelRawToClusterCUDA::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
   edm::ParameterSetDescription desc;
   desc.add<bool>("isRun2", true);
+  desc.add<bool>("isHIon", false);
   desc.add<bool>("IncludeErrors", true);
   desc.add<bool>("UseQualityInfo", false);
+  desc.add<uint32_t>("MaxWord", pixelgpudetails::MAX_WORD);
```

Comment:
could you change `MaxWord` to `MaxFEDWords` to clarify the intent of the option ?